docs/immortal-empires/kislev.md

```diff
@@ -0,0 +1,31 @@
+# Kislev
+
+## Kostaltyn: The Great Orthodoxy
+
+* **Route I - There Can Only Be One**
+  * We are weak alone, and stronger together. Unite under my banner, support me, and I will give you my strength.
+    * Erengrad, Praag, Kislev
+  * Have at least 600 Supporters
+  * **Reward:**
+    * Right and Just Leadership: Public Order +4, Growth +60
```

Comment:
M: Add the Motherland cooldown effect. 
W: `:` Between Bonus and amount -> `Public Order: +4, Growth: +60`. At least that is how default effects are.